docs/Users_Guide/roc_diagram.rst

```diff
@@ -51,27 +52,103 @@ METplus Configuration
 
 .. literalinclude:: ../../metplotpy/plots/config/roc_diagram_defaults.yaml
 
+
+
+Modify the `stat_input` setting (input data file) in this default config file to ensure that the data file is accessible.
+Open the $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/config/roc_diagram_defaults.yaml file and replace
+the relative path in the stat_input entry `stat_input:  ../../../test/roc_diagram/plot_20200507_074426.data`
+with the path to your $METPLOTPY_SOURCE directory.  For example, if your $METPLOTPY_SOURCE is /username/plotting,
+then the `stat_input` setting will look like the following:
+
+`stat_input:  /username/plotting/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+
 **Custom Configuration File**
 
+
 A second, `mandatory` configuration file is required, which is
-used to customize the settings to the ROC diagram plot. The custom_reliability.yaml
+used to customize the settings to the ROC diagram plot. The custom_roc_diagram.yaml
 file is included with the source code and looks like the following:
 
-.. literalinclude:: ../../metplotpy/plots/roc_diagram/custom_roc_diagram.yaml
+.. literalinclude:: ../../test/roc_diagram/custom_roc_diagram.yaml
+
+copy this custom config file from the directory where you saved the source code to your working directory:
+
+``cp $METPLOTPY_SOURCE/METplotpy/test/roc_diagram/custom_roc_diagram.yaml $WORKING_DIR/custom_roc_diagram.yaml``
+
+
+Modify the `stat_input` and `plot_filename` settings in the
+$METPLOTPY_SOURCE/METplotpy/test/roc_diagram/custom_roc_diagram.yaml
+file to explicitly point to the $METPLOTPY_SOURCE/METplotpy/test/roc_diagram/roc_diagram directory (where
+the custom config files and sample data reside).  Replace the relative path `./plot_20200507_074426.data`
+with the full path `$WORKING_DIR/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+For example:
+
+`stat_input: /username/working_dir/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+`plot_filename: /username/working_dir/output_plots/roc_diagram_custom.png`
+
+where $WORKING_DIR was replaced with the actual name of the directory.  In this example, the
+/username/working_dir/output_plots directory exists and has the appropriate read and write permissions.
+
+
+If you wish to save the intermediate `.points1` file (used by METviewer but also useful for debugging), set the following configuration values:
+
+`dump_points_1 setting: 'True'`
 
-If the user
-wishes to use all the default settings defined in the roc_diagram_defaults.yaml
-file, an empty custom configuration file (minimal_roc_diagram_defaults.yaml)
-can be specified instead:
+And add this line (anywhere in the file) if it doesn't already exist, to set the optional points_path:
 
-.. literalinclude:: ../../metplotpy/plots/roc_diagram/minimal_roc_diagram.yaml
+`points_path: '/dir_to_save_points1_file'`
 
 
+Replace the `'/dir_to_save_points1_file'` to the directory where you wish to save the file.
```

Comment:
```suggestion
Replace the `'/dir_to_save_points1_file'` to the directory where you wish to save the file.  If points_path is commented out (indicated by a '#' symbol in front of it), remove the '#' symbol to uncomment the points_path so that it will be used by the code.
```